testing/polybar/APKBUILD

```diff
@@ -0,0 +1,92 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=polybar
+pkgver=3.3.0
+pkgrel=0
+pkgdesc="A fast and easy-to-use tool for creating status bars."
+url="https://polybar.github.io/"
+arch="all"
+license="MIT"
+makedepends="
+	alsa-lib-dev
+	cairo-dev
+	cmake
+	curl-dev
+	i3wm
+	jsoncpp-dev
+	libmpdclient-dev
+	libnl3-dev
+	libxcb-dev
+	pulseaudio-dev
+	python2
+	xcb-proto
+	xcb-util-cursor-dev
+	xcb-util-image-dev
+	xcb-util-renderutil-dev
+	xcb-util-wm-dev
+	xcb-util-xrm-dev
+	"
+subpackages="
+	$pkgname-doc
+	$pkgname-bash-completion:bashcomp:noarch
+	$pkgname-zsh-completion:zshcomp:noarch
+	"
+install="
+	$pkgname.post-install
+	"
+_xppver="56fbf8459809ba65f9ea4a65b8f467a6e9890c4e"
+_i3ipcppver="0.7.1"
+source="
+	$pkgname-$pkgver.tar.gz::https://github.com/jaagr/polybar/archive/$pkgver.tar.gz
+	xpp-$_xppver.tar.gz::https://github.com/jaagr/xpp/archive/$_xppver.tar.gz
+	https://github.com/jaagr/i3ipcpp/archive/v$_i3ipcppver.tar.gz
+	"
+builddir="$srcdir/$pkgname-$pkgver"
+prepare() {
+	cd "$builddir"
+	mkdir -p build
+	rm -rf lib/xpp
+	mv "$srcdir"/xpp-$_xppver lib/xpp
+	rm -rf lib/i3ipcpp
+	mv "$srcdir"/i3ipcpp-$_i3ipcppver lib/i3ipcpp
+	default_prepare
+}
+
+build() {
+	cd "$builddir"/build
+	cmake -DCMAKE_INSTALL_PREFIX=/usr ..
+	cmake --build .
+}
+
+check() {
+	cd "$builddir"
+	# At least we can do this without X
+	./build/bin/polybar -v > /dev/null 2>&1
```

Comment:
Just running `polybar -v` is not appropriate as it often gives very little coverage.